### DISCUSSION TITLE
Aliases are listed upon starting terminal of after every command is executed

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [marlonrichert/zsh-autocomplete#195] [Q&A] [answered]
I'm not sure if this is a new feature or not but after upgrading to release `2021-02-25`, whenever I start the terminal or after every command, my aliases are listed beneath the prompt like so:

<img width="715" alt="Screenshot 2021-02-27 at 20 32 19" src="https://user-images.githubusercontent.com/219075/109399480-f925cc00-793a-11eb-9d10-3c98756403de.png">

**Comment by cibinmathew** [comment, marked as answer]
may be try this to let the auto suggestions appear only if n min chars
`zstyle ':autocomplete:*' min-input 2`

Additionally if with a delay 
`zstyle ':autocomplete:*' min-delay .2  # seconds`

**Reply by Pilgrim1379** [reply]
Thanks @cibinmathew. I already had the delay settings in my config but that wasn't doing it however changing to `min-input` option fixed it. Thanks.
